models/public/yolo-v1-tiny-tf/yolo-v1-tiny-tf.md

```diff
@@ -89,10 +89,10 @@ The array of detection summary info, name - `conv2d_9/BiasAdd`,  shape - `1,13,1
 - `Cx`, `Cy` - cell index
 
 Detection box has format [`x`,`y`,`h`,`w`,`conf`,`class_no_1`, ..., `class_no_20`], where:
-- (`x`,`y`) - coordinates of box center, relative to cell
-- `h`,`w` - normalized height and width of box
-- `conf` - confidence of detection box
-- `class_no_1`,...,`class_no_20` - score for each class in logits format
+- (`x`,`y`) - raw coordinates of box center, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get coordinates relative to the cell 
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get height and width values relative to cell
+- `conf` - confidence of detection box, use [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
```

Comment:
This line uses "use" where the previous used "apply". It would be better to be consistent.

Reply:
Thanks! This is done now